exercises/nucleotide-count/src/test/java/NucleotideCounterTest.java

```diff
@@ -13,18 +13,11 @@
     @Rule
     public ExpectedException expectedException = ExpectedException.none();
 
-    @Test
-    public void testEmptyDnaStringHasNoAdenine() {
-        NucleotideCounter nucleotideCounter = new NucleotideCounter("");
-        assertThat(nucleotideCounter.count('A'), is(0));
-    }
-
     @Ignore("Remove to run test")
```

Comment:
Since this is now the first test it shouldn't have an `@Ignore` statement above it 🙂